### ISSUE TITLE
The panel content doesn't response to onTap event after drag to expand

### ISSUE THREAD

**Issue by Warpath** [RaviKavaiya/sliding_panel#19]
Hi, RaviKavaiya
 The panel content doesn't respond to the first onTap event after drag closed to expanded.  Once u scroll the list or tap again it triggers.

```
import 'package:flutter/material.dart';
import 'package:sliding_panel/sliding_panel.dart';

class TestWidget extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return SlidingPanel(
                    panelController: PanelController(),
                    initialState: InitialPanelState.closed,
                    size: PanelSize(expandedHeight: 0.85),
                    autoSizing: PanelAutoSizing(headerSizeIsClosed: true),
                    duration: const Duration(milliseconds: 300),
                    backdropConfig: BackdropConfig(enabled: false),
                    decoration: PanelDecoration(
                        backgroundColor: Colors.white,
                        borderRadius: const BorderRadius.only(
                            topLeft: Radius.circular(8),
                            topRight: Radius.circular(8))),
                    backPressBehavior: BackPressBehavior.COLLAPSE_POP,
                    content: PanelContent(
                      panelContent: _buildChildList(),
                      headerWidget: PanelHeaderWidget(
                        headerContent: Container(
                          height: 100,
                          color: Colors.green,
                        ),
                        decoration: PanelDecoration(
                            backgroundColor: Colors.white,
                            borderRadius: const BorderRadius.only(
                                topLeft: Radius.circular(8),
                                topRight: Radius.circular(8))),
                      ),
                    ));
  }

  List<Widget> _buildChildList() {
    List<Widget> list = List();
    for (int i = 0; i < 100; i++) {
      list.add(GestureDetector(
        onTap: () {
          print('click $i');
        },
        child: Container(
          height: 70,
          child: Text('$i'),
        ),
      ));
    }
    return list;
  }
  
  
}
```

**Comment by RaviKavaiya** [comment]
Hi @Warpath I investigated about this issue. Thanks for reporting...
Until the problem gets resolved, I would request you to use this workaround:
`snapping: PanelSnapping.forced`

**Comment by RaviKavaiya** [comment]
@Warpath This issue has been fixed in `v1.2.4`. Please check it out. You can close this issue now.

Once again, thanks for the help...

**Comment by Warpath** [comment]
@RaviKavaiya Thanks for the quick update, the problem has been solved. Regards
